src/chainstate/burn/db/sortdb.rs

```diff
@@ -619,6 +645,67 @@ impl <'a> SortitionHandleTx <'a> {
         self.set_stacks_block_accepted_at_tip(&chain_tip, consensus_hash, parent_stacks_block_hash, stacks_block_hash, stacks_block_height)
     }
 
+    fn pick_recipients(&self, reward_set_vrf_seed: &SortitionHash, next_pox_info: Option<&RewardCycleInfo>) -> Result<Option<RewardSetInfo>, BurnchainError> {
+        if let Some(next_pox_info) = next_pox_info {
+            if let PoxAnchorBlockStatus::SELECTED_AND_KNOWN(ref anchor_block, ref reward_set) = next_pox_info.anchor_status {
```

Comment:
`SELECTED_AND_KNOWN` or `SelectedAndKnown`?

Reply:
Probably `SelectedAndKnown`

Reply:
Updated in 4a6a38a